app/lib/report_helper.rb

```diff
@@ -19,6 +19,16 @@ def report_details(report)
     }
   end
 
+  def view_level_name2int(view_level)
+    case view_level.downcase
```

Comment:
should all the functions really be here or it should be in app/helpers ?

Reply:
Should this specific function use TaxonCount::TAX_LEVEL_SPECIES and TaxonCount::TAX_LEVEL_GENUS constants?   I assume those are available without SQL queries?

Reply:
Yes, I'll change to use TaxonCount::TAX_LEVEL_SPECIES etc